### PR TITLE
i18zh-cn: Upgrade & Migrate -> Upgrade from Tauri 1.0

### PR DESCRIPTION
**Pull request by Wind-Explorer**
#### What kind of changes does this PR include?

- Translated content for Tauri 2.0 docs

#### Description
- zh-cn translation of Upgrade from Tauri 1.0 page under Upgrade & Migrate page.

### CODE REVIEW

**Comment by lorenzolewis** [comment]
Thanks for this! @dklassic would you be able to help out with a review on this one?

File: src/content/docs/zh-CN/guides/upgrade-migrate/from-tauri-1.mdx [hunk 1]
```diff
@@ -0,0 +1,1150 @@
...
+            .check("很nb")
+            .build()?;
+        let menu = MenuBuilder::new(app).item(&submenu).build()?;
+        Ok(())
+    })
+```
+
+`tauri::Builder::menu` now takes a closure because the menu needs a Manager instance to be built. See [the documentation](https://docs.rs/tauri/2.0.0-alpha/tauri/struct.Builder.html#method.menu) for more information.
+
+`tauri::Builder::menu` 现在接受一个闭包，因为菜单需要构建一个 Manager 实例。 有关更多信息，请参阅[文档](https://docs.rs/tauri/2.0.0-alpha/tauri/struct.Builder.html#method.menu)。
```
Comment (dklassic) [inline src/content/docs/zh-CN/guides/upgrade-migrate/from-tauri-1.mdx]:
```suggestion
`tauri::Builder::menu` 现在接受一个闭包，因为菜单需要构建一个 Manager 实例。 有关更多信息，请参阅[文档](https://docs.rs/tauri/2.0.0-alpha/tauri/struct.Builder.html#method.menu)。
```

**Review by dklassic — CHANGES_REQUESTED** [review]
Thanks for contributing!

I find the translation to be pretty good, but besides on particular direct change suggestion, I'd like to ask you to do another sweep to improve the reading experience:

- Ensure a space is in between Chinese characters and half sized characters.
- Remove redundant space between two full sized characters (both Chinese characters and punctuations).
- Properly replace half sized punctuation with full sized Chinese punctuations, e.g. use "：" instead of ": ".

Also regarding inline translation of technical term coexisting with the original term, for example in the case of "迁移 Path 路径 至 Tauri Manager 管理器" the formatting is rather confusing to read. I would suggest to use brackets to the mark translated term, i.e. "迁移 Path（路径）至 Tauri Manager（管理器）"

**Comment by Wind-Explorer** [comment]
Seems I'm a bit late to the party, translations for this particular page has been completed by #1967. Closing.

**Pull request closed without merging**